tests/run-pass/fs.rs

```diff
@@ -82,6 +82,19 @@ fn main() {
     // Removing file should succeed.
     remove_file(&path).unwrap();
 
+    // Renaming a file should succeed.
+    let path1 = tmp.join("rename_source.txt");
+    let path2 = tmp.join("rename_destination.txt");
+    // Clean files for robustness.
+    remove_file(&path1).ok();
+    remove_file(&path2).ok();
+    let file = File::create(&path1).unwrap();
+    drop(file);
+    rename(&path1, &path2).unwrap();
+    assert_eq!(ErrorKind::NotFound, path1.metadata().unwrap_err().kind());
+    assert!(path2.metadata().unwrap().is_file());
+    remove_file(&path2).ok();
```

Comment:
```suggestion
    remove_file(&path2).unwrap();
```
This should always succeed, right?

Reply:
Good point, updated